### PR TITLE
Unify SSH runner in installer_api_test.py

### PR DESCRIPTION
**Pull request by mellenburg**
Replace parallel implementation of one-off SSH
communication with ssh/ssh_tunnel.py module

### CODE REVIEW

**Comment by malnick** [comment]
Thank you for making this one a bit smaller :)

File: test_util/installer_api_test.py [hunk 1]
```diff
@@ -3,58 +3,77 @@
...
-                '-oBatchMode=yes',
-                '-oPasswordAuthentication=no']
-
-        def ssh(cmd):
-            assert isinstance(cmd, list)
-            return ['/usr/bin/ssh']+ssh_opts+['{}@{}'.format(ssh_user, host)]+cmd
-
-        def scp(src, dst):
-            return ['/usr/bin/scp']+ssh_opts+[src, '{}@{}:{}'.format(ssh_user, host, dst)]
+        if not tunnel:
```
Comment (malnick) [inline test_util/installer_api_test.py]: Nit: can we have `if tunnel; else` and flip the blocks? The negative logic is ever so slightly more complicated to read, but it's in no way a blocker.

Comment (mellenburg) [inline test_util/installer_api_test.py]: Definitely!

**Comment by mellenburg** [comment]
@cmaloney 
@spahl 
This code is ready for reivew

**Comment by spahl** [comment]
:ship:

**Pull request merged**